ibc/docs/README.md

```diff
@@ -0,0 +1,77 @@
+# IBC <!-- omit in toc -->
+
+- [Overview](#overview)
+- [IBC Module](#ibc-module)
+  - [Host Configuration](#host-configuration)
+- [Components](#components)
+  - [ICS-24 Host Requirements](#ics-24-host-requirements)
+
+## Overview
+
+![IBC High Level](./high-level-ibc.png)
+
+Inter-Blockchain Communication (IBC) is a protocol that facilitates the trustless communication between two chains. This allows them to interact with each other via the relaying of IBC packets. This works by having two IBC enabled chains: **chain A** and **chain B**; both of which will have a light client for the other running on their network. In order for **chain A** to transfer some of their native token to **chain B** a few things must be in place: a connection between the two chains, a channel and port must be opened between them on this connection, a light client for the opposing chain on each end of the connection and finally a relayer is needed to actually transfer the packet. Once all these are in place, **chain A** can commit an IBC packet to its state containing the information around the transaction, and generate a proof detailing its inclusion in the state at the specified height. The relayer will then submit this proof to **chain B** which will verify the proof and if valid **chain B** can then react to this packet in an appropriate fashion (ie. sending an IBC **chain A** token to the specified address on **chain B**).
+
+## IBC Module
+
+Pocket's IBC module is split into numerous components detailed below. The overall module layout is as follows:
+
+**Note:** Not all of the different ICS components have been fully implemented yet, this is a work in progress.
+
+```mermaid
+flowchart TB
+    subgraph IBC[IBC Module]
+        subgraph 23[ICS-23]
+            VC[Vector Commitments]
+        end
+        subgraph 24[ICS-24]
+            HR[Host Requirements]
+        end
+        subgraph I[IBC Interface]
+            IF[ICS-25 Handler Interface]
+            IR[ICS-26 Routing Module]
+        end
+    end
+    subgraph 2[ICS-02]
+        CL[Client Semantics]
+    end
+    subgraph 3[ICS-03]
+        CO[Connection Semantics]
+    end
+    subgraph 5[ICS-05]
+        PA[Port Allocation]
+    end
+    subgraph 4[ICS-04]
+        CP[Channel & Packet Semantics]
+    end
+    subgraph 20[ICS-20]
+        FT[Fungible Token Transfer]
+    end
+    IBC ---> 2
+    IBC ---> 3
+    IBC ---> 5
+    IBC ---> 4
+    IBC ---> 20
+```
+
+### Host Configuration
```

Comment:
Wdyt about "node" instead of "host" here to be consistent with other Pocket docs (and the line below)?

Reply:
> My observation was that the content under this section seems to be describing how to configure a pocket node to be an IBC host instead.

Yeah I understand now, will edit accordingly